src/umpire/strategy/FixedPool.cpp

```diff
@@ -190,5 +190,19 @@ FixedPool::numPools() const noexcept
   return m_pool.size();
 }
 
+bool 
+FixedPool::pointerIsFromPool(void* ptr) const noexcept
+{
+  for (auto& p : m_pool) {
+    const char* t_ptr = reinterpret_cast<char*>(ptr);
```

Comment:
To get pedantic, this could use the uniform initialization syntax we've recently introduced in the code.

Reply:
We can do this in a separate PR the next time FixedPool is changed.